tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -405,6 +405,9 @@
       ,
       {
         "name": "zipkin",
+        {{ if .tracing_tls }}
+        "tls_context": {{ .tracing_tls }},
```

Comment:
tls_context is deprecated, we should use transport_socket

Reply:
can be followup

Reply:
done